sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_RecognizeHealthcareEntities.md

```diff
@@ -0,0 +1,104 @@
+
+await healthOperation.WaitForCompletionAsync();
+
+RecognizeHealthcareEntitiesResultCollection results = healthOperation.Value;
+
+Console.WriteLine($"Results of Azure Text Analytics \"Healthcare Async\" Model, version: \"{results.ModelVersion}\"");
+Console.WriteLine("");
+
+foreach (DocumentHealthcareResult result in results)
+{
+		Console.WriteLine($"    Recognized the following {result.Entities.Count} healthcare entities:");
+
+		foreach (HealthcareEntity entity in result.Entities)
+		{
+				Console.WriteLine($"    Entity: {entity.Text}");
+				Console.WriteLine($"    Subcategory: {entity.Subcategory}");
+				Console.WriteLine($"    Offset: {entity.Offset}");
+				Console.WriteLine($"    Length: {entity.Length}");
+				Console.WriteLine($"    IsNegated: {entity.IsNegated}");
+				Console.WriteLine($"    Links:");
+
+				foreach (HealthcareEntityLink healthcareEntityLink in entity.Links)
+				{
+						Console.WriteLine($"        ID: {healthcareEntityLink.Id}");
+						Console.WriteLine($"        DataSource: {healthcareEntityLink.DataSource}");
+				}
+		}
+		Console.WriteLine("");
+}
+}
+```
+
+## Recognizing healthcare entities in multiple documents
+
+To recognize healthcare entities in multiple documents, call `StartHealthcareBatchAsync` on an `IEnumerable` of strings.  The result is a Long Running operation of type `HealthcareOperation` which polls for the results from the API.
+
+```C# Snippet:TextAnalyticsSampleRecognizeHealthcare
+string document = "Subject is taking 100mg of ibuprofen twice daily.";
+
+HealthcareOperation healthOperation = await client.StartHealthcareAsync(document);
+
+await healthOperation.WaitForCompletionAsync();
+
+RecognizeHealthcareEntitiesResultCollection results = healthOperation.Value;
+```
+
+To recognize healthcare entities in a collection of documents in different languages, call `RecognizeHealthcare EntitiesBatch` on an `IEnumerable` of `TextDocumentInput` objects, setting the `Language` on each document.
+
+```C# Snippet:TextAnalyticsSampleRecognizeHealthcare
+var documents = new List<TextDocumentInput>
+{
+    new TextDocumentInput("1", "Subject is taking 100mg of ibuprofen twice daily.")
+    {
+         Language = "en",
+    },
+    new TextDocumentInput("2", "Can cause rapid or irregular heartbeat.")
+    {
+         Language = "en",
+    },
+    new TextDocumentInput("3", "The patient is a 54-year-old gentleman with a history of progressive angina over the past several months")
+    {
+         Language = "en",
+    }
+};
+
+HealthcareOperation healthOperation = await client.StartHealthcareAsync(document);
+
+await healthOperation.WaitForCompletionAsync();
+
+RecognizeHealthcareEntitiesResultCollection results = healthOperation.Value;
+```
+
+To see the full example source files, see:
+
+* [Synchronously RecognizeHealthcare ](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_Healthcare.cs)
```

Comment:
This looks great!
again, if you see CI failing because of the links, then you can comment it out